### ISSUE TITLE
Version 2.0 Task List

### ISSUE THREAD

**Issue by aero31aero** [jbt/markdown-editor#77]
**TLDR** - This project is going to be actively developed again, and the issues for the major overhaul coming soon are open [here](https://github.com/aero31aero/markdown-editor/issues?q=is%3Aissue+is%3Aopen+label%3Amerge-blocker). You can see the most likely final version [here](http://rohitt.me/markdown-editor).

Hello all. I'm the new maintainer of this project. I was earlier maintaining a fork of this project, however, since my fork got quite farther from the original it isn't feasible for me to create a huge PR now. This is why I have merged some of the PRs on this repository to my fork at https://github.com/aero31aero/markdown-editor manually. Example: #60, #59.

Currently, I'm doing final migrations and have created some issues in that repository that need to be resolved before I hard reset `jbt/markdown-editor/master` to `aero31aero/markdown-editor/master`. Please see the current version at http://rohitt.me/markdown-editor as it progresses and feel free to open any other merge blocking issues you encounter with that version.

[This](https://github.com/aero31aero/markdown-editor/issues?q=is%3Aissue+is%3Aopen+label%3Amerge-blocker) is the list of issues that I plan to resolve before a hard reset. About the currently open PRs to this repo, I plan to take care to cherry-pick them onto the new HEAD after the reset. After that, the development would proceed back to how it was done earlier.

If you have any qualms with my plan, please drop a comment below and let's discuss the alternatives.

Regards
Rohitt

**Comment by aero31aero** [comment]
Version 2 is live now. Development shall continue on this repository now. Ping @jbt. Would love to have your views on this.
